picard/coverart/__init__.py

```diff
@@ -130,7 +137,7 @@ def download_next_in_queue(self):
                     p = provider(self)
                     if p.enabled():
                         log.debug("Trying cover art provider %s ..." % name)
-                        ret = p.queue_downloads()
+                        ret = p.queue_images()
```

Comment:
Hmmm ;)
Should i revert the rename ? Or fixing plugins is easy enough ? We don't have an api version for this.